docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -177,11 +177,19 @@ children:
         url: core-tutorial-part11.html
         output: 'web, pdf'
 
-- title: 'Usage Scenarios'
+- title: 'How-to guides'
```

Comment:
I think this change should be extracted into a standalone PR, where we should also update `Usage-senarios.html` to something like `how-to-guides.html` while keeping the old URL as an alias to avoid broken (404) links.

Reply:
The page "Usage-scenarios.html" was removed by https://github.com/strongloop/loopback-next/pull/5773.

I opened  #5799 to rename the sidebar section.